production/helm/loki/templates/backend/service-backend.yaml

```
@@ -34,4 +34,7 @@ spec:
       protocol: TCP
   selector:
     {{- include "loki.backendSelectorLabels" . | nindent 4 }}
+{{- if .Values.backend.service.trafficDistribution }}
```

I guess it would be great, if this can be define once as well.

```suggestion
{{- with .Values.backend.service.trafficDistribution | default .Values.loki.service.trafficDistribution }}
```

Please add `Values.loki.service.trafficDistribution` to the values as well.

---

Added @jkroepke !